contrib/management-packs/isilon-onefs-mpack/src/main/resources/addon-services/ONEFS/1.0.0/kerberos.json

```diff
@@ -26,7 +26,8 @@
           "core-site": {
             "hadoop.security.authentication": "kerberos",
             "hadoop.security.authorization": "true",
-            "hadoop.proxyuser.HTTP.groups": "${hadoop-env/proxyuser_group}"
+            "hadoop.proxyuser.HTTP.groups": "${hadoop-env/proxyuser_group}",
```

Comment:
This is generally better off in the service advisor so you can append to an existing value.

Reply:
This came from the kerberos.json of HDFS. I left it there but onefs might not even need it.